test/models/report_test.rb

```diff
@@ -3,7 +3,16 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'editable?' do
+    report = reports(:"alice's_report")
+    he = users(:bob)
+    me = users(:alice)
+    assert_not report.editable?(he)
+    assert report.editable?(me)
+  end
+  test 'created_on' do
+    user = users(:alice)
+    report = user.reports.create!(title: '夏', content: '暑い')
+    assert_equal Time.zone.now.to_date, report.created_on
```

Comment:
```suggestion
    assert_equal report.created_at.to_date, report.created_on
```

createしたタイミングとnowの時刻がずれる可能性があるので、上のように書いた方がより確実ですー

Reply:
`report.created_at`に直しました。